data/battle_scripts_1.s

```diff
@@ -2047,11 +2047,11 @@ BattleScript_AlreadyAtFullHp::
 
 BattleScript_EffectFakeOut::
 	attackcanceler
-	jumpifnotfirstturn BattleScript_ButItFailedAtkStringPpReduce
+	jumpifnotfirstturn BattleScript_ButItFailedPpReduceAtkString
 	setmoveeffect MOVE_EFFECT_FLINCH | MOVE_EFFECT_CERTAIN
 	goto BattleScript_EffectHit
 
-BattleScript_ButItFailedAtkStringPpReduce::
+BattleScript_ButItFailedPpReduceAtkString::
```

Comment:
All these names are clunky and literal, I don't think the name in the PR is objectively better. The current name at least has the benefit of listing the commands in the order they occur.

Reply:
Then I wonder if it would be better if we do something similar to what `BattleScript_HitFrom` does
![image](https://github.com/pret/pokeemerald/assets/2904965/2a1bbe3b-3a3d-462a-a367-f65f034ef548)

Reply:
That sounds good to me. It's basically what I suggested, but with an extra "From" to make things even clearer :eyes: